SharedPackages/VPN/Sources/VPN/NetworkProtectionOptionKey.swift

```diff
@@ -26,8 +26,10 @@ public enum NetworkProtectionOptionKey {
     public static let dnsSettings = "dnsSettings"
     public static let excludeLocalNetworks = "excludeLocalNetworks"
     public static let isAuthV2Enabled = "isAuthV2Enabled"
+#if os(macOS)
```

Comment:
We don't use these values in iOS - I'd rather have them guarded for now so it's clearer there's mis-alignment across paltforms.